src/index/diskann_zparameters.h

```diff
@@ -15,21 +15,22 @@
 
 #pragma once
 
+#include <distance.h>
+
 #include <string>
```

Comment:
no need empty line

Reply:
done

Reply:
the empty line is automatically added by clang-format